core/modules/layout/plugins/renderers/layout_renderer_standard.inc

```diff
@@ -146,14 +154,21 @@ class LayoutRendererStandard {
    */
   function __construct(Layout $layout, array $renderer_plugin) {
     $this->plugin = $renderer_plugin;
-    $layout_info = layout_get_layout_template_info($layout->layout_template);
+    $layout_template_info = layout_get_layout_template_info($layout->layout_template);
     $this->layout = &$layout;
 
-    if (empty($layout_info)) {
+    if (empty($layout_template_info)) {
       throw new LayoutMissingException(t('The layout plugin "@plugin" could not been found to display the "@title" layout.', array('@plugin' => $layout->layout_template, '@title' => $layout->title)));
     }
     else {
-      $this->layout_info = $layout_info;
+      $this->layout_template_info = $layout_template_info;
+    }
+
+    // Adjust for the renaming of "layout_info" to "layout_template_info".
+    // See https://github.com/backdrop/backdrop-issues/issues/6628.
+    // @todo: Remove in 2.x.
+    if (isset($this->layout_info)) {
+      $this->layout_info = $this->layout_template_info;
     }
```

Comment:
Same thing here, I think this `if` statement would prevent `$this->layout_info` from ever being populated.

```suggestion
    // Adjust for the renaming of "layout_info" to "layout_template_info".
    // See https://github.com/backdrop/backdrop-issues/issues/6628.
    // @todo: Remove in 2.x.
    $this->layout_info = $this->layout_template_info;
```